lib/anyplayer/selector.rb

```diff
@@ -71,21 +69,21 @@ def player_load(player)
 
     def player_on_platform?(player)
       result = player.platforms.include?(platform)
-      log "  not on platform" if !result
+      log '  not on platform' unless result
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.